scrapers/PuffyNetworkSite.yml

```
@@ -0,0 +1,36 @@
+name: PuffyNetworkSite
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - simplyanal.com
+      - weliketosuck.com
+      - wetandpissy.com
+      - wetandpuffy.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="header-title"]/h2
+        postProcess:
+          - replace:
+            - regex: ^(.*)[—]\s+
+              with:
+      Date:
+        selector: //span[@class="date-movie"]/b
+        postProcess:
+          - parseDate: Jan 2, 2006
+      Details: //div[@class="movie-description"]/text()[1]
+      Tags:
+        Name: //div[@class='tags-box']//a/text()
+      Performers:
+        Name: //span[@class="model-movie"]//a/text()
+      Studio:
+        Name:
+          selector: //img[@class="logo-overley"]/@src|//ul[@class="nav-list"]/li/a/img/@src
+          postProcess:
+            - replace:
+              - regex: .+?\.(.+?)\..+
+                with: $1
```

```suggestion
          postProcess:
            - replace:
                - regex: .+?\.(.+?)\..+
                  with: $1
            - map:
                weliketosuck: We Like To Suck
                wetandpuffy: Wet And Puffy
                wetandpissy: Wet And Pissy
                simplyanal: Simply Anal
```
The above change keep the studio names consistent with the parent studio scraper